### PR TITLE
Ana-Peykova

### PR DESCRIPTION
**Pull request by anapeykova**

### CODE REVIEW

File: 2023/spring/Scheffler-MW@0230PM/ana-peykova.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Hi there, I'm **Ana**!
+## (and this is a very informative subtitle)
+
+### About Me
+*[Insert a brief introduction or overview of the person's life and accomplishments]*
+
+### Two truths and a lie
```
Comment (AbdullahKhurram30) [inline 2023/spring/Scheffler-MW@0230PM/ana-peykova.md:7]: Try adding a new line for both.
The easiest fix is just an additional line between the 2 different sentences.

File: 2023/spring/Scheffler-MW@0230PM/ana-peykova.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+> I love CS162
```
Comment (AbdullahKhurram30) [inline 2023/spring/Scheffler-MW@0230PM/ana-peykova.md:8]:
```suggestion
> I love CS162

```

File: 2023/spring/Scheffler-MW@0230PM/ana-peykova.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
...
+> I'm a cat person
```
Comment (AbdullahKhurram30) [inline 2023/spring/Scheffler-MW@0230PM/ana-peykova.md:9]:
```suggestion
> I'm a cat person

```

**Review by AbdullahKhurram30 — CHANGES_REQUESTED** [review]

**Review by Aromzyy — APPROVED** [review]
LGTM

**Pull request closed without merging**